docs/section11/Rule_11-9.md

```diff
@@ -0,0 +1,35 @@
+# Service_Water_Heating - Rule 11-9
+**Schema Version:** 0.0.38  
+
+**Mandatory Rule:** FALSE
+
+**Rule ID:** 11-9
+
+**Rule Description:** "The baseline system must be sized according to Standard 90.1 2019, Section 7.4.1."  
+
+**Rule Assertion:** UNDETERMINED / NOT_APPLICABLE
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, a
+
+**Evaluation Context:** B-RMD each SHW BAT
+**Data Lookup:**   
+**Function Call:** 
+- **get_spaces_associated_with_each_SWH_bat**
+
+**Applicability Checks:**
+- check that the SHW BAT in the B_RMD has SHW loads
+
+## Applicability Checks:
+- only projects with SHW for the SHW space BAT in the baseline model are applicable
+
+- use the function get_spaces_associated_with_each_SWH_bat to get a list of spaces for each SHW BAT: `shw_and_spaces_dict = get_spaces_associated_with_each_SWH_bat(U_RMD)`
```

Comment:
My opinion - 
~~use the function get_spaces_associated_with_each_SWH_bat to~~ get a list of spaces for each SHW BAT: 
is easier to follow. let the pseudo code part be the reference for calling the function name